### PR TITLE
docs: how to organize your project for tilt

### PR DESCRIPTION
**Pull request by nicks**
Hello @lianmakesthings, @landism,

Please review the following commits I made in branch nicks/multi-repo:

67b87664164d005c7740342f6ebc8e36a60b0136 (2022-02-15 14:41:42 -0500)
docs: how to organize your project for tilt

Code review reminders, by giving a LGTM you attest that:

* Commits are adequately tested
* Code is easy to understand and conforms to style guides
* Incomplete code is marked with TODOs
* Code is suitably instrumented with logging and metrics

### CODE REVIEW

**Comment by nicks** [comment]
I tried to reorganize this doc to be less about "here's a specific solution" vs "here's a bunch of different patterns from the first thing people hack in to get things working, to more complex multi-repo management tooling"

File: docs/multiple_repos.md [hunk 1]
```diff
@@ -1,16 +1,160 @@
...
+to specify sets of services to run or to edit. For more examples, see the
+[per-user config guide](tiltfile_config.html).
+
+## Managing other git Repositories
+
+The simplest way to handle multi-repository projects is to ask the user to check
+out other repos as a sibling directory.
+
+```python
+if os.path.exists('../backend'):
```
Comment (nicksieger) [inline docs/multiple_repos.md:117]:
```suggestion
if not os.path.exists('../backend'):
```

File: docs/multiple_repos.md [hunk 2]
```diff
@@ -1,16 +1,160 @@
...
+  fail('Please "git clone" the backend repo in ../backend!')
+  
+include('../backend/Tiltfile')
+```
+
+You can also supply the repository as an environment variable.
+
+```python
+backend_dir = os.environ.get('BACKEND_REPO_DIR', '../backend')
+if os.path.exists(backend_dir):
```
Comment (nicksieger) [inline docs/multiple_repos.md:127]:
```suggestion
if not os.path.exists(backend_dir):
```

File: docs/multiple_repos.md [hunk 3]
```diff
@@ -1,16 +1,160 @@
...
-Each team or developer may have different needs in starting specific services, especially if your entrypoint Tiltfile is pointing to many downstream Tiltfiles. Consider using [Tiltfile Configs](tiltfile_config.html) to help customize accordingly.
+## How to Organize Your Project for Tilt
 
-If you have Tiltfile logic that can be abstracted out of your specific system, consider using [Extensions](extensions.html).
+Tilt does not require a particular project structure. We try to provide 
+APIs to help you pull the services you need into your dev environment, no matter
+where they're defined.
 
+Do NOT restructure your project for Tilt. Instead, restructure your Tiltfiles
+to match your project structure!
```
Comment (lianmakesthings) [inline docs/multiple_repos.md:19]:
```suggestion
>> Do NOT restructure your project for Tilt. Instead, restructure your Tiltfiles to match your project structure!
```
Suggest to put this in a blockquote to make it more visible

Screenshot 2022-02-16 at 13 36 53

**Review by lianmakesthings — APPROVED** [review]
This is great! Thanks for updating the guide 🙏

**Pull request merged**